README.md

```diff
@@ -37,6 +37,7 @@ colcon build
 ```
 
 ## Run
+The TURTLEBOT3_MODEL environment variable must be set when running the simulation application (not needed for robot application). Valid values are burger, waffle, and waffle_pi.
```

Comment:
Nitpick: Maybe enclose them in ticks, so they appear as `burger`, `waffle`, and `waffle_pi`.